cmake/compiler/vs2019.cmake

```
@@ -15,20 +15,35 @@ else()
   set (VS2019_VARIABLES_ 1)
 endif()
 
-set(ROOT "/c")
+set(ROOT "c:")
```

Windows drive letters are traditionally capital letters

```suggestion
set(ROOT "C:")
```